docs/source/usage.rst

```diff
@@ -95,6 +98,22 @@ to conserve storage space: ::
 This exclude pattern will skip all restart subdirectories under the short-term archive,
 except for those with years ending in '0' or '5'.
 
+Example with Globus
+-------------------
+If you run zstash on the system without the HPSS file system, but has a [Globus](https://globus.org) endpoint set up,
+you can use a Globus URL: ::
+
+  $ cd $CSCRATCH/ACME_simulations/20170731.F20TR.ne30_ne30.anvil
+  $ zstash create --hpss=globus://9cd89cfd-6d04-11e5-ba46-22000b92c6ec/~/test/E3SM_simulations/20170731.F20TR.ne30_ne30.anvil
```

Comment:
`$ zstash create --hpss=globus://9cd89cfd-6d04-11e5-ba46-22000b92c6ec/~/test/E3SM_simulations/20170731.F20TR.ne30_ne30.anvil <local path>`

Reply:
Corrected, https://github.com/E3SM-Project/zstash/pull/154/commits/9ceff8c78a5e22bf8ee070312ea299ec8b898c72. Instead of <local path>, I added a dot '.', so the two Globus examples are consisted with other examples above.